src/test/java/ProfileTest.java

```diff
@@ -0,0 +1,37 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import Pages.ProfilePage;
+import org.testng.annotations.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+
+public class ProfileTest extends BaseTest{
+
+    @Test(priority = 0)
+    public void changeProfileName()throws InterruptedException{
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.